.ci/scripts/shellcheck

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+set -e
+docker run --rm -v "$PWD:/mnt" koalaman/shellcheck:stable "$@"
```

Comment:
is this image trustworthy?

Reply:
It is used in several Editors (Sublime, Atom, VS, ...)https://github.com/koalaman/shellcheck#in-your-editor, It is installed on several CI by default (Travis CI and others), it has a plugin for SonarQube, and has a package for Debian https://packages.debian.org/stretch/shellcheck. so it is pretty extend used.